### PR TITLE
MM-28226: Admin Advisor: add mitigations

### PR DESCRIPTION
**Pull request by catalintomai**
#### Summary
Add various mitigations for Admin Advisor metric warning job.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28226

#### Related Pull Requests
https://github.com/mattermost/mattermost-server/pull/15379

### CODE REVIEW

File: components/announcement_bar/configuration_bar/configuration_bar.jsx [hunk 1]
```diff
@@ -181,7 +181,7 @@ class ConfigurationAnnouncementBar extends React.PureComponent {
 
                     return (
                         <AnnouncementBar
-                            showCloseButton={true}
+                            showCloseButton={false}
```
Comment (fmunshi) [inline components/announcement_bar/configuration_bar/configuration_bar.jsx:184]: Not sure what this announcement bar is for but do we really want an announcement bar that cannot be closed?

Comment (fmunshi) [reply components/announcement_bar/configuration_bar/configuration_bar.jsx:184]: Just read the ticket, I guess we do

Comment (catalintomai) [reply components/announcement_bar/configuration_bar/configuration_bar.jsx:184]: Yeah, I guess the thinking is that makes the user open the learn more and read the message in the modal.

**Review by srkgupta — CHANGES_REQUESTED** [review]
Hi @catalintomai 

I am seeing the following issues:

1. The styling of the Acknowledge button does not matches up with the expected screenshot added in the ticket. The expected is to look like this:
Screenshot 2020-09-02 at 11 00 55 PM

But actually the button looks like this:
Screenshot 2020-09-02 at 11 00 47 PM


2) Banner X button is still displayed. As per the point 4 in the ticket https://mattermost.atlassian.net/browse/MM-28226, the Banner X button should be removed. 

Screenshot 2020-09-02 at 11 08 05 PM

**Comment by catalintomai** [comment]
> Hi @catalintomai
> 
> I am seeing the following issues:
> 
> 1. The styling of the Acknowledge button does not matches up with the expected screenshot added in the ticket. The expected is to look like this:
> 
> Screenshot 2020-09-02 at 11 00 55 PM
> 
> But actually the button looks like this:
> Screenshot 2020-09-02 at 11 00 47 PM
> 
> 1. Banner X button is still displayed. As per the point 4 in the ticket https://mattermost.atlassian.net/browse/MM-28226, the Banner X button should be removed.
> 
> Screenshot 2020-09-02 at 11 08 05 PM

@srkgupta - something seems to be amiss with your setup - could it be that there is some caching issue? The button's styling hasn't changed and the X button has been definitely turned off - see code change.

**Review by srkgupta — APPROVED** [review]
Tested the updated PR and all the issues are fixed now and the PR is working fine as expected. Approving the PR.

**Pull request merged**